tests/unit/modules/test_chocolatey.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+'''
+Test for the chocolatey module
+'''
+
+# Import Python libs
+from __future__ import absolute_import
+import os
+
+# Import Salt Libs
+import salt.modules.chocolatey as chocolatey
+import salt.utils
+
+# Import Salt Testing Libs
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.unit import TestCase, skipIf
+from tests.support.mock import MagicMock, patch
+
+if salt.utils.is_windows():
+    CHOCO_PATH = 'C:\\path\\to\\chocolatey.exe'
+    CHOCO_PATH_PD = os.path.join(
+        os.environ.get('ProgramData'), 'Chocolatey', 'bin', 'chocolatey.exe')
+    CHOCO_PATH_SD = os.path.join(
+        os.environ.get('SystemDrive'), 'Chocolatey', 'bin', 'chocolatey.bat')
+
+    MOCK_FALSE = MagicMock(return_value=False)
```

Comment:
The rest of the globals are OK, but this mock object should be initialized in the test class, `setUpClass` for example.

Reply:
Ideally, all globals :)

Reply:
Done